products/apache-activemq.md

```diff
@@ -31,11 +31,11 @@ auto:
 # - y <= 9 and z <= 9 : "https://activemq.apache.org/activemq-{{'__LATEST__'|replace:'.','00'}}-release"
 releases:
 -   releaseCycle: "6.1"
-    releaseDate: 2024-03-11
+    releaseDate: 2024-04-15
     eol: false # still listed on https://activemq.apache.org/components/classic/download/
     latest: "6.1.2"
-    latestReleaseDate: 2024-04-11
-    link: https://activemq.apache.org/components/classic/download/classic-06-01-00
+    latestReleaseDate: 2024-04-15
```

Comment:
we're using the git tag date (https://github.com/apache/activemq/releases/tag/activemq-6.1.2) to automate this and simplify updates. Even if it's sometime not exact, it's pretty close and much more manageable for the maintainers.

So unfortunately this change is likely to be reverted next time there is an auto-update.